src/app/cart/cart.component.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles.scss';
+
+.modal{
+    .modal-header{
+        background-color: $color-black;
+        color: $color-white;
+    }
+    .modal-body{
+        height: 500px;
+        background-color: $color-gray;
+        .cart-body{
+            text-align: center;
+            
+        }
+    }
+}
+
+.cart-offer{
+    margin: 10px;
+    display: flex;
+    .offer-img{
+        flex: 1;
+    }
+    .offer-text{
+        flex: 1;
+    }
+}
+
+.icon-fa{
+    margin: 0 10px;
+    color: $color-pink;
+    font-size: 20px;
+}
+
+.item-price{
+margin: 0 5px;
```

Comment:
Please fix the formatting issues (all files)

Reply:
done